Makefile

```diff
@@ -120,3 +122,14 @@ run:
 	sed -i "s,file: brokers/.*.yml,file: brokers/${MF_BROKER_TYPE}.yml," docker/docker-compose.yml
 	sed -i "s,MF_BROKER_URL=.*,MF_BROKER_URL=$$\{MF_$(shell echo ${MF_BROKER_TYPE} | tr 'a-z' 'A-Z')_URL\}," docker/.env
 	docker-compose -f docker/docker-compose.yml up
+
+endtoend:
+	cd scripts && . ./end-to-end.sh
+
+rundetached:
```

Comment:
Do not add this target, rather inspect the argument given to `run` target, or pre-set ENV variable when calling make (for example `D=1 make run`) - something like [this](https://stackoverflow.com/questions/2214575/passing-arguments-to-make-run) or [this](https://chriswiegman.com/2021/08/using-parameters-in-a-make-target/)

Reply:
Added $(ARGS)
```
run:
	sed -i "s,file: brokers/.*.yml,file: brokers/${MF_BROKER_TYPE}.yml," docker/docker-compose.yml
	sed -i "s,MF_BROKER_URL=.*,MF_BROKER_URL=$$\{MF_$(shell echo ${MF_BROKER_TYPE} | tr 'a-z' 'A-Z')_URL\}," docker/.env
	docker-compose -f docker/docker-compose.yml up $(ARGS)
```
to run in detached: `make run ARGS="-d`
to run normally: `make run`